O21.Game/Engine/ReproducibleRandom.fs

```diff
@@ -29,16 +32,16 @@ type ReproducibleRandom private (backend: Random, idGenerator: SequentialIdGener
         backend.Next 100 >= 50
         
     member _.NextFishId() =
-        idGenerator.GetFishId()
+        idGenerator.NextIdWithPrefix(FishId.prefix) |> FishId
         
     member _.NextBombId() =
-        idGenerator.GetBombId()
+        idGenerator.NextIdWithPrefix(BombId.prefix) |> BombId
         
     member _.NextBonusId() =
-        idGenerator.GetBonusId()
+        idGenerator.NextIdWithPrefix(BonusId.prefix) |> BonusId
```

Comment:
Honestly I don't understand why these methods were introduced in the `ReproducibleRandom` class to begin with; they are not supposed to be random at all.

Let's not touch these and `SequentialIdGenerator` in this PR? I'll open a separate issue to move them out of this type.

The `SequentialIdGenerator` was supposed to be _sequential_, not _random_. Moreover, random by its very definition is not _unique_, and uniqueness is what we need from the identifiers.

Reply:
#319.